app/serializers/index.js

```diff
@@ -0,0 +1,14 @@
+exports.serializeUserOutput = user => ({
+  id: user.id,
+  first_name: user.firstName,
+  last_name: user.lastName,
+  email: user.email
+});
+
+exports.serializeUserInput = user => ({
```

Comment:
this would be a mapper https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#28--mappers

Reply:
user mapper added